e2e-tests/cypress/fixtures/application.js

```diff
@@ -77,6 +84,13 @@ const application = {
     [ESTIMATED_ANNUAL_TURNOVER]: '65000',
     [PERCENTAGE_TURNOVER]: '0',
   },
+  EXPORTER_BROKER: {
+    [NAME]: 'name',
+    [ADDRESS_LINE_1]: 'Address line 1',
+    [TOWN]: 'town',
+    [POSTCODE]: 'SW1A 2AA',
+    [EMAIL]: 'test@test.com',
```

Comment:
Shall we use `Cypress.env('GOV_NOTIFY_EMAIL_RECIPIENT')`? Just so we're not hard coding emails